src/table/utils/handle-key-press.ts

```diff
@@ -187,16 +191,20 @@ export const handleBodyKeyDown = ({
 }: HandleBodyKeyDownProps) => {
   const isSelectionMode = selectionsAPI.isModal();
   // Adjust the cellCoord depending on the totals position
-  const firstBodyRowIdx = totalsPosition === 'top' ? 2 : 1;
-  const cellCoord: [number, number] = [cell.rawRowIdx + firstBodyRowIdx, cell.rawColIdx];
+  const rowIdxDiff = totalsPosition === 'top' ? 2 : 1;
```

Comment:
I'm not sure this name is better, is the old one not clear?

Reply:
yes, sure. I had another idea first then changed it but forgot to undo the name.